Update Rich Schema HIPEs: make `issuer` and `issuanceDate` default attributes for any Mapping (#158)
hyperledger/indy-hipe: text/0155-rich-schema-mapping/README.md

as the corresponding Rich Schema.
A Mapping may contain only a subset of the original Rich Schema's attributes.

> Every Mapping must have two default attributes required by any W3C compatible credential
>  (see [W3C verifiable credential specification](https://www.w3.org/TR/vc-data-model)): `issuer` and `issuanceDate`.
> This allows to selectively disclose these attributes in the same way as other attributes from the schema.  

Reviewer: ```suggestion
Every Mapping must have two default attributes required by any W3C compatible credential
 (see [W3C verifiable credential specification](https://www.w3.org/TR/vc-data-model)): `issuer` and `issuanceDate`.
 Additionally, any other attributes that are considered optional by the
[W3C verifiable credential specification](https://www.w3.org/TR/vc-data-model) that will be included in the
issued credential must be included in the Mapping. For example, `credentialStatus` or `expirationDate`.
This allows the holder to selectively disclose these attributes in the same way as other attributes from the schema.  
```

#### attributes
A dict of all the schema attributes the Mapping object is going to map to encodings and use in credentials.
An attribute may have nested attributes matching the schema structure. 

> In addition to the schema attributes, it must contain the following default attributes required by any W3C compatible
> verifiable credential: 
>  - `issuer`
>  - `issuanceDate`

Reviewer: ```suggestion
It must also contain the following default attributes required by any W3C compatible
verifiable credential (plus any additional attributes that may have been included from the
W3C verifiable credentials data model): 
 - `issuer`
 - `issuanceDate`
 - any additional attributes
```

    '@type': "rdfs:Class",
    "schema": "did:sov:4e9F8ZmxuvDqRiqqY29x6dx9oU4qwFTkPbDpWtwGbdUsrCD",
    "attribuites" : {
        "issuer": [{
            "enc": "did:sov:9x9F8ZmxuvDqRiqqY29x6dx9oU4qwFTkPbDpWtwGbdUsrCD",
            "rank": 1
        }],

>         "issuanceDate": [{
>             "enc": "did:sov:119F8ZmxuvDqRiqqY29x6dx9oU4qwFTkPbDpWtwGbdUsrCD",
>             "rank": 2
>         }],    

Reviewer: ```suggestion
        "issuanceDate": [{
            "enc": "did:sov:119F8ZmxuvDqRiqqY29x6dx9oU4qwFTkPbDpWtwGbdUsrCD",
            "rank": 2
        }],    
        "expirationDate": [{
            "enc": "did:sov:119F8ZmxuvDqRiqqY29x6dx9oU4qwFTkPbDpWtwGbdUsrCD",
            "rank": 11
        }],            
```

    - `schema`:  An `id` of the corresponding Rich Schema
    - `attributes` (dict): A dict of all the schema attributes the Mapping object is going to map to encodings and use in credentials.
                    An attribute may have nested attributes matching the schema structure. 
                    In addition to the schema attributes, it must contain the following default attributes required by any W3C compatible

>                     verifiable credential:
>                     
>         - `issuer`
>         - `issuanceDate`

Reviewer: ```suggestion
                    verifiable credential (plus any additional attributes that may have been included from the
W3C verifiable credentials data model):
                    
        - `issuer`
        - `issuanceDate`
        - any additional attributes
```
